client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/SymptomsFields/useSymptomsFields.ts

```diff
@@ -0,0 +1,35 @@
+import { SweetAlertResult } from 'sweetalert2';
+
+import theme from 'styles/theme';
+import useCustomSwal from 'commons/CustomSwal/useCustomSwal';
+import React from "react";
+
+const useSymptomsFields = (params: useSymptomsFieldsInCome): useSymptomsFieldsOutCome => {
+    const { alertWarning } = useCustomSwal();
+    const { didSymptomsDateChangeOccur, setDidSymptomsDateChangeOccur} = params;
+
+    const handleSymptomsDateDataChange = () => {
```

Comment:
Why there is double "Date" on the name?

Reply:
it's date data
יענו מידע על תאריכים כי צריך לעשות את הבדיקה הזאת לא רק אם התאריך של הסימפטומים משתנה אלא גם אם יש סימפטומים והאם תאריך סימפטומים ידוע

Reply:
My bad. Didn't see it.